qutebrowser/completion/models/miscmodels.py

```diff
@@ -112,6 +112,10 @@ def delete_buffer(data):
 
     model = completionmodel.CompletionModel(column_widths=(6, 40, 54))
 
+    # abbreviate tabs_are_windows for convenience
+    taw = True if config.val.tabs.tabs_are_windows else False
```

Comment:
I don't think you need `True if ... else False` here, given that you already have a boolean - this basically boils down to `True if True else False` or `True if False else False` :wink:

Also, `taw` is a bit too short IMHO and makes the rest unnecessarily hard to read - why not call it `tabs_are_windows`? Then you probably don't need the comment anymore either.